Loading conda
Loading cuda version 12.6
Loading cudnn version 8.9.7.29-12
Traceback (most recent call last):
  File "main_cd.py", line 83, in <module>
    train(args)
  File "main_cd.py", line 15, in train
    model.train_models()
  File "/home/asadhgauri_umass_edu/deforestation_detection/changeformer/ChangeFormer/models/trainer.py", line 306, in train_models
    self._load_checkpoint()
  File "/home/asadhgauri_umass_edu/deforestation_detection/changeformer/ChangeFormer/models/trainer.py", line 168, in _load_checkpoint
    self.net_G.load_state_dict(torch.load(self.args.pretrain), strict=False)
  File "/work/pi_jaimedavila_umass_edu/asadhgauri_umass_edu/.conda/envs/ChangeFormer/lib/python3.8/site-packages/torch/serialization.py", line 594, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/work/pi_jaimedavila_umass_edu/asadhgauri_umass_edu/.conda/envs/ChangeFormer/lib/python3.8/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/work/pi_jaimedavila_umass_edu/asadhgauri_umass_edu/.conda/envs/ChangeFormer/lib/python3.8/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'None'
